bears/go/GoMetaLintBear.py

```diff
@@ -0,0 +1,66 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.GoRequirement import GoRequirement
+from coalib.settings.Setting import typed_list
+
+
+@linter(executable='gometalinter',
+        output_format='regex',
+        output_regex=r'(?P<filename>\w+\.go):'
```

Comment:
capturing filename this emits a warning, because the filename is always constant

Reply:
You have not created a GlobalBear , so that is all irrelevant.